test/e2e/conformance_test.go

```diff
@@ -134,6 +137,15 @@ func TestClusterConformance(t *testing.T) {
 			// Create infrastructure
 			t.Log("Provisioning infrastructure using Terraform…")
 			args := []string{}
+			img, err := DiscoverControlPlaneOSImage(tc.provider, testOperatingSystem)
+			if err != nil {
+				t.Fatalf("failed to discover control plane os image: %v", err)
+			}
+			// Custom images are only supported for AWS currently
+			if img != "" {
+				args = append(args, "-var", img)
+				args = append(args, "-var", fmt.Sprintf("worker_os=%s", testOperatingSystem))
```

Comment:
Worker OS shouldn't depend on custom image availability, I mean, making them independent makes more sense for whole test suite (will allow to test non-ununtu worker os on all providers).

Reply:
Fixed. You can now specify both control plane and workers OS